webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -8,6 +8,7 @@ import toasts from './toasts';
 import { reducers as passwordStrengthReducers } from '../../components/PasswordStrength';
 import { reducers as breadcrumbBarReducers } from '../../components/BreadcrumbBar';
 import { reducers as autoCompleteReducers } from '../../components/AutoComplete';
+import { reducers as dualListReducers } from '../../components/DualList';
```

Comment:
early thought, just something I have been thinking for a while: with redux, what is going to happen, when we got two components of the same kind on the page? :thinking: Will they not start to fight over the same namescpace in redux state tree?

Reply:
great thought!
We could take advantage of an ID mechanism, which will be the namespace for the reducer.
I open a PR to [distinguish between autocumpletes](https://github.com/theforeman/foreman/pull/6501) which we can implement here too !